src/server.c

```diff
@@ -2878,7 +2878,10 @@ void initServer(void) {
     }
 
     scriptingInit(1);
-    functionsInit();
```

Comment:
functionsInit will return C_ERR when error. It’s just a supplement and can be discarded